opennms/values.yaml

```diff
@@ -131,6 +131,7 @@ opennms:
     storage: # Sizes for RRDs and configuration
       etc: 1Gi # Warning Google Filestore enforces 1TB at a minimum
       rrd: 1000Gi # Depends on how many files you are planning to have (each installation is different)
+      mibs: null
```

Comment:
Perhaps it is worth adding a comment that size value is expected here (like `1Gi`).

Reply:
Fixed.